pkg/oc/bootstrap/docker/up.go

```diff
@@ -366,9 +366,13 @@ func (c *CommonStartConfig) Complete(f *osclientcmd.Factory, cmd *cobra.Command,
 	c.dockerClient = client
 	taskPrinter.Success()
 
-	// TODO HOLY SIDE-EFFECTS this is setting global bool values as part of a check!
 	// Check that we have the minimum Docker version available to run OpenShift
 	taskPrinter.StartTask("Checking Docker version")
+	// Indicate whether the Docker is Red Hat
+	c.isRHDocker, err = c.DockerHelper().IsRedHat()
```

Comment:
Do we care about this value or can we directly set that nsenter thing?  I suspect we can remove the entire struct value and use this to compute the value for `useNsenterMount`

Reply:
yes, I got rid of isRHDocker field